[2025-03-12T15:05:39.168+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-12T15:05:39.261+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: validate_all_tables.products scheduled__2025-03-05T13:00:00+00:00 [queued]>
[2025-03-12T15:05:39.300+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: validate_all_tables.products scheduled__2025-03-05T13:00:00+00:00 [queued]>
[2025-03-12T15:05:39.304+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-12T15:05:39.365+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): products> on 2025-03-05 13:00:00+00:00
[2025-03-12T15:05:39.383+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1314) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-12T15:05:39.391+0000] {standard_task_runner.py:72} INFO - Started process 1326 to run task
[2025-03-12T15:05:39.389+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'validate_all_tables', 'products', 'scheduled__2025-03-05T13:00:00+00:00', '--job-id', '488', '--raw', '--subdir', 'DAGS_FOLDER/s3_data.py', '--cfg-path', '/tmp/tmpknd0hf2r']
[2025-03-12T15:05:39.423+0000] {standard_task_runner.py:105} INFO - Job 488: Subtask products
[2025-03-12T15:05:39.804+0000] {task_command.py:467} INFO - Running <TaskInstance: validate_all_tables.products scheduled__2025-03-05T13:00:00+00:00 [running]> on host b404e9833d11
[2025-03-12T15:05:40.240+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='validate_all_tables' AIRFLOW_CTX_TASK_ID='products' AIRFLOW_CTX_EXECUTION_DATE='2025-03-05T13:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-05T13:00:00+00:00'
[2025-03-12T15:05:40.256+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-12T15:05:40.258+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-12T15:05:40.261+0000] {logging_mixin.py:190} INFO - Current task name:products state:running start_date:2025-03-12 15:05:39.262676+00:00
[2025-03-12T15:05:40.262+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables and current dag run status:running
[2025-03-12T15:05:40.263+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-12T15:05:41.022+0000] {httpchecksum.py:481} INFO - Skipping checksum validation. Response did not contain one of the following algorithms: ['crc32', 'sha1', 'sha256'].
[2025-03-12T15:05:41.217+0000] {logging_mixin.py:190} INFO - Validation successful. Found 321 product records.
[2025-03-12T15:05:41.218+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-12T15:05:41.241+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-12T15:05:41.242+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=validate_all_tables, task_id=products, run_id=scheduled__2025-03-05T13:00:00+00:00, execution_date=20250305T130000, start_date=20250312T150539, end_date=20250312T150541
[2025-03-12T15:05:41.315+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-12T15:05:41.316+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-12T15:05:41.317+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables queued_at:2025-03-12 15:04:59.240294+00:00
[2025-03-12T15:05:41.318+0000] {logging_mixin.py:190} INFO - Task hostname:b404e9833d11 operator:PythonOperator
[2025-03-12T15:05:41.346+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-12T15:05:41.486+0000] {taskinstance.py:3901} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-03-12T15:05:41.490+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
